sklearn/naive_bayes.py

```diff
@@ -412,8 +413,12 @@ def _update_class_log_prior(self, class_prior=None):
             self.class_log_prior_ = np.log(class_prior)
         elif self.fit_prior:
             # empirical prior, with sample_weight taken into account
-            self.class_log_prior_ = (np.log(self.class_count_)
-                                     - np.log(self.class_count_.sum()))
+            class_count = np.where(self.class_count_ != 0, 
```

Comment:
I have no idea if this is a good fix. How is ``self.class_count_`` zero? Is that for ``partial_fit``?